pages/integrations/azure.md

```diff
@@ -0,0 +1,87 @@
+# [Microsoft Azure IoT Suite][azure] integration
+
+[Resin.io][resin] makes it simple to deploy, update, and maintain code running on remote devices. Microsoft's new IoT Suite makes it easy to manage and capture the data those devices generate. This tutorial will walk you through using the two in conjuction. 
+
+## Connect your device to resin.io
+
+Follow our [getting started guide][installing] or watch our screencast to get your device connect to your resin.io dashboard. Once your device shows up on the dashboard your ready for the next step.
+
+![factory-build](/img/integrations/azure/factory-build.png)
+
+## Create the resin.io API Key 
+
+Find your Application ID in the resin.io dashboard url: https://dashboard.resin.io/apps/NNNN/devices and your auth-token from the [preferences panel](https://dashboard.resin.io/preferences?tab=details). Then combine insert the two into the curl request below.
+
+```
+curl -H 'Authorization: Bearer AUTH_TOKEN' -X POST https://api.resin.io/application/NNNN/generate-api-key
+```
+
+This will return your new API key. 
+
+__NOTE:__ The key will be returned in quotation marks, but these should be stripped before using it in the following step.
+
+## Add resin.io API key to IoT Hub Admin portal 
+
+Go to the IoT Hub Admin portal select 'Resin.io Config' and set the `App ID` and the newly generated `API Key`.
+
+![IoT-hub-creds](/img/integrations/azure/iot-hub-creds.png)
+
+This will automatically create a new device on IoT Hub with every device you currently have on resin.io as well as every future device you may have, as it constantly polls for new devices.
+
+![devices-pending](/img/integrations/azure/devices-pending.png)
+
+This also automatically creates application wide and per device environment variables on resin.io that are accessible by your code. eg. `$IOT_HUB_DEVICE_IOT`. This obviously simplifies sending data to the IoT hub.
+
+![resin-envar](/img/integrations/azure/envar.png)
+
+## Push the sample app to your devices
+
+When you push code to resin git endpoint several things happen, including:
+* resin creates a docker container
+* it provisions it following the instructions in Dockerfile (for example, it automatically installs Linux packages and builds C sources in our case)
+* it notifies the devices about the new container availability
+
+When you power on the device resin-agent connects to resin API server and fetches the application container. It also does it every time a new container is available (when you push the updated code).
+
+First clone the the sample application to your local machine.
+
+__NOTE:__ Our integration will not be part of the code microsoft releases, instead it will stay as a separate fork that should be used instead of the official sample solution.
```

Comment:
So I see where this note ended up being, but this is the case for _both_ the sample app and the Admin Portal that's called the azure-iot-solution.